policy/modules/services/mon.te

```diff
@@ -161,9 +169,10 @@ optional_policy(`
 #
 
 # sys_ptrace is for reading /proc/1/maps etc
-allow mon_local_test_t self:capability { sys_ptrace sys_admin };
+allow mon_local_test_t self:capability { dac_override dac_read_search setgid setuid sys_rawio sys_ptrace sys_admin };
```

Comment:
The `storage_raw_read_fixed_disk` should have the `sys_rawio` with it, so I'd rather keep it there and drop it here, unless there is another reason to need the perm.

Reply:
done